games.py

```diff
@@ -1,9 +1,13 @@
 """Games, or Adversarial Search. (Chapter 5)
 """
 
+import collections
+import math
+import random
+
 from utils import *  # noqa
 
-import random
+infinity = math.inf
```

Comment:
@norvig Was that a mistake? Because I think infinity is a nice variable to have in utils.py
I can't think of any reason to remove it.